examples/nixnet/CAN_SIGNAL_SINGLE_POINT_OUTPUT.py

```diff
@@ -0,0 +1,87 @@
+
+import grpc
+import nixnet_pb2 as nixnet_types
+import nixnet_pb2_grpc as grpc_nixnet
+import getch
+
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+
+# Parameters
+INTERFACE = "CAN1"
+DATABASE = "NIXNET_example"
+CLUSTER = "CAN_Cluster"
+SIGNAL_LIST = "CANEventSignal1,CANEventSignal2"
+NUM_SIGNALS = 2
+SUCCESS = 0
+
+# Display Error Function
+def displayerror_andexit(Status, Source):
+    status_string = [None] * 1024
+    nixnet_types.StatustoString(Status,status_string.__sizeof__(),Source) 
```

Comment:
Spaces after `,` while passing arguments to `StatusToString`. Something like `nixnet_types.StatustoString(Status, status_string.__sizeof__(), Source)`